CONTRIBUTING.md

```diff
@@ -126,6 +128,25 @@ We use several GitHub apps to help maintain this repository. While we would like
 
 Currently, we use the [Stale](https://github.com/apps/stale) and [Lock](https://github.com/apps/lock) apps to mark old issues as stale, and to lock issues which have been closed to stop drive-by comments. You can see the configuration files for these in [.github/](.github).
 
+## Generating the CONTRIBUTORS.md file
```

Comment:
Are we actually doing this right now?

Reply:
We should be! :)

Reply:
Now with the @all-contributors bot installed, maybe that’s obsolete?

Reply:
Well, it wasn't when this PR was started. 😆